templates/taxi/manufacturer_list.html

```diff
@@ -21,30 +24,31 @@ <h1>
       {% for manufacturer in manufacturer_list %}
         <tr>
           <td>
-              {{ manufacturer.id }}
+            {{ manufacturer.id }}
           </td>
           <td>
-              {{ manufacturer.name }}
+            {{ manufacturer.name }}
+            <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">
+              Update
+            </a>
           </td>
           <td>
-              {{ manufacturer.country }}
+            {{ manufacturer.name }}
           </td>
           <td>
-              <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">
-                Update
-              </a>
-            </td>
-            <td>
-              <a style="color: red"
-                href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}">
-                Delete
-              </a>
-            </td>
+            {{ manufacturer.name }}
```

Comment:
The `name` and `country` fields are repeated multiple times in the table row. Ensure that each column displays the correct and unique information. For example, line 36 should display `{{ manufacturer.country }}` instead of `{{ manufacturer.name }}`.